### PR TITLE
Most recently viewed movies

### PR DESCRIPTION
**Pull request by x3igh7**
Tests passed!

### CODE REVIEW

File: spec/models/recent_movies_spec.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+require 'spec_helper'
+
+describe RecentMovies do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```
Comment (linstula) [inline spec/models/recent_movies_spec.rb:6]: delete this extra file

File: spec/controllers/users_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+require 'spec_helper'
+
+describe UsersController do
+
+end
```
Comment (linstula) [inline spec/controllers/users_controller_spec.rb]: delete this extra file

File: app/views/reviews/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+<h1><%= @review.title %></h1>
+<h3><%= @review.body %></h3>
```
Comment (linstula) [inline app/views/reviews/show.html.erb:2]: Jason doesnt like new line errors (add an extra line to the end of this file)

File: db/schema.rb [hunk 4]
```diff
@@ -41,6 +41,20 @@
 
   add_index "rails_admin_histories", ["item", "table", "month", "year"], :name => "index_rails_admin_histories"
 
+  create_table "recent_movies", :force => true do |t|
+    t.integer  "user_id"
```
Comment (wannabefro) [inline db/schema.rb]: null => false on the database

File: app/models/recent_review.rb [hunk 5]
```diff
@@ -0,0 +1,7 @@
+class RecentReview < ActiveRecord::Base
+  attr_accessible :user, :review
+
+  belongs_to :user
+  belongs_to :review
```
Comment (wannabefro) [inline app/models/recent_review.rb:7]: validates_presence_of :user, :review

File: spec/features/user_profile_spec.rb [hunk 6]
```diff
@@ -0,0 +1,58 @@
+require 'spec_helper'
+require 'pry'
```
Comment (jzgit) [inline spec/features/user_profile_spec.rb]: you have pry in your gemfile so you dont have to rerequire it here

File: spec/features/user_profile_spec.rb [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+       visit movie_review_path(movie, review)
+       expect(RecentReview.count).to eq(previous + 1)
+       visit root_path
+       visit movie_review_path(movie, review) 
+       expect(RecentReview.count).to eq(previous + 1)
+     end
+    end
+
+  context "Movies" do
+    it 'shows the most recently visted movie' do
```
Comment (jzgit) [inline spec/features/user_profile_spec.rb:55]: you should visit a couple movies and asset that they are displayed in the correct order here

File: spec/features/user_profile_spec.rb [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+
+  it "shows the current user's information" do
+    visit user_path(valid_user)
+
+    expect(page).to have_content("Profile for", valid_user.email)
+  end
+
+  context "Reviews" do
+
+    it 'shows the most recently visted movie reviews' do
```
Comment (jzgit) [inline spec/features/user_profile_spec.rb:20]: you should visit a couple movie reivews and asset that they are displayed in the correct order and only the last 3 are displayed

**Comment by jzgit** [comment]
fix the test with the comments i've added and then you can merge

**Pull request merged**